homeassistant/components/telegram_bot/__init__.py

```diff
@@ -176,12 +182,18 @@
 SERVICE_SCHEMA_SEND_FILE = BASE_SERVICE_SCHEMA.extend(
     {
         vol.Optional(ATTR_URL): cv.template,
+        vol.Optional(ATTR_THUMB_URL): cv.template,
+        vol.Optional(ATTR_THUMB_FILE): cv.template,
         vol.Optional(ATTR_FILE): cv.template,
         vol.Optional(ATTR_CAPTION): cv.template,
         vol.Optional(ATTR_USERNAME): cv.string,
         vol.Optional(ATTR_PASSWORD): cv.string,
         vol.Optional(ATTR_AUTHENTICATION): cv.string,
         vol.Optional(ATTR_VERIFY_SSL): cv.boolean,
+        vol.Optional(ATTR_SUPPORTS_STREAMING): cv.boolean,
+        vol.Optional(ATTR_DURATION): cv.positive_int,
+        vol.Optional(ATTR_WIDTH): cv.positive_int,
+        vol.Optional(ATTR_HEIGHT): cv.positive_int,
```

Comment:
In the docs, you wrote that the following attributes are required if `ATTR_SUPPORTS_STREAMING` is set. Please add validation for it.